.github/workflows/main.yml

```
@@ -0,0 +1,22 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install dependencies
+        run: npm ci
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

[actions has its own token](https://github.com/marketplace/actions/create-a-release)